samples/HTML+Django/nunjucks.njk

```diff
@@ -0,0 +1,9 @@
+{% extends "parent.html" %}
```

Comment:
Is this from a "real" application? Do you have a more realistic sample file? Sample files are used to train our Bayesian classifier so it's important they reflect in-the-wild syntaxes and keywords.